src/api/app/models/token/workflow.rb

```diff
@@ -31,11 +31,6 @@ def call(options)
     raise Token::Errors::SCMTokenInvalid, e.message
   end
 
-  # Only used by rebuild steps
-  def package_find_options
-    { use_source: false, follow_project_links: true, follow_multibuild: true }
```

Comment:
You noticed that they differ per token type right?

Reply:
Yes, thats why I moved it from the workflow token model to the individual steps, in order to pass the correct options for the Triggerable concern